egs/swbd/s5c/local/chain/run_tdnn_4q.sh

```diff
@@ -17,7 +17,7 @@ stage=10
 train_stage=-10
 get_egs_stage=-10
 speed_perturb=true
-dir=exp/chain/tdnn_4q  # Note: _sp will get added to this if $speed_perturb == true.
+dir=exp/chain/tdnn_4q_trial2  # Note: _sp will get added to this if $speed_perturb == true.
```

Comment:
I would suggest waiting till few iterations are done. 
I had copied this recipe from a branch with a lot of other changes, which is the reason for these errors.